C++/eggpuzzle.cpp

```diff
@@ -0,0 +1,50 @@
+/* Contributed by Dhrubajyoti Chakraborty */
+
+/* Egg Drop Puzzle Solution */
+
+#include <bits/stdc++.h> 
```

Comment:
Please remove extra spaces from the code,

Reply:
fixed